bp/src/BPConfiguration.m

```diff
@@ -557,50 +558,6 @@ - (BOOL)parseXcTestRunFile:(NSString *)path withError:(NSError *__autoreleasing
     return YES;
 }
 
-- (BOOL)parseXcSchemeFile:(NSString *)schemePath withError:(NSError *__autoreleasing *)errPtr {
```

Comment:
where is this big chunk of code now ?

Reply:
Gone! Parsing the `xcscheme` files was a mistake. We now support two modes of operation: `xctestrun` files for Xcode compatibility and `.app` + `.xctest` which just loads them from the app. The _hack_ of parsing the `.xcscheme` files was to get command line args and environment for the app. Now bluepill will correctly get them from the config file or `.xctestrun` file in all cases.